functorch/csrc/BatchRulesScatterOps.cpp

```diff
@@ -254,14 +258,36 @@ std::tuple<Tensor,optional<int64_t>> scatter_src_batch_rule(
   src_ = ensure_has_bdim(src_, src_bdim.has_value(), batch_size);
   auto physical_dim = getPhysicalDim(self_, /*has_batch_dim*/true, dim);
 
-  auto result = at::scatter(self_, physical_dim, index_, src_);
+  auto result = f(self_, physical_dim, index_, src_);
   // result should have same shape as self
   if (self_logical_rank == 0) {
     result = result.squeeze(-1);
   }
   return std::make_tuple(result, 0);
 }
 
+} // namespace
+
+std::tuple<Tensor,optional<int64_t>> scatter_src_batch_rule(
+    const Tensor& self, optional<int64_t> self_bdim,
+    int64_t dim,
+    const Tensor& index, optional<int64_t> index_bdim,
+    const Tensor& src, optional<int64_t> src_bdim) {
+  using scatter_sig = Tensor (*)(const Tensor&, int64_t, const Tensor&, const Tensor&);
+  return scatter_batch_rule(static_cast<scatter_sig>(at::scatter),
```

Comment:
nit: `static_cast<scatter_sig>(at::scatter)` can be replaced with `ATEN_FN2(scatter, src)`. The ATEN_FN macros make it so that we don't need to manually type out the signatuers